pkg/common/utils/utils.go

```diff
@@ -77,27 +79,27 @@ func GeneralPodFilter(pod *v1.Pod) bool {
 
 func GetQueueNameFromPod(pod *v1.Pod) string {
 	queueName := constants.ApplicationDefaultQueue
-	if an, ok := pod.Labels[constants.LabelQueueName]; ok {
+	if an := GetPodLabelValue(pod, constants.LabelQueueName); an != "" {
```

Comment:
cc @wilfred-s Here we originally prefer label first, if we need to change to prefer AnnotationQueueName first?